### ISSUE TITLE
quantization tool failed at eva.generate_quantized_model (AIV-552)

### ISSUE THREAD

**Issue by skodippily** [espressif/esp-dl#103]
Hello,
Error while executing the line below in [example.py](https://github.com/espressif/esp-dl/blob/master/tools/quantization_tool/examples/example.py). 
 `eva.generate_quantized_model(model_proto, pickle_file_path)`
I slightly modify example.py to load yolov2tiny model to it.

error:
![image](https://user-images.githubusercontent.com/41458359/192676510-ae17bb86-982e-4980-809e-38cefcbf54d4.png)

Optimized model file satisfied all the layers defined in the link [here](https://github.com/espressif/esp-dl/tree/master/include/layer)

Could you please help me with this problem.

**Comment by Auroragan** [comment]
HI, what is scalarPreprocessor_scale/bias? Can you find them in the onnx graph?

**Comment by skodippily** [comment]
Yes, those are under MUL and ADD layers
![image](https://user-images.githubusercontent.com/41458359/194998668-b2c4518c-2ed0-439a-b4cf-d37a8db9364e.png)

I'm trying to do a project base on object detection using ESP-DL. Most of the pre-trained models are not supported with built-in operators right? So the only model that I could find is this [tinyyolov2](https://github.com/onnx/models/blob/main/vision/object_detection_segmentation/tiny-yolov2/model/tinyyolov2-7.onnx) model.  If you know any other pre-trained model supported ESP-DL, please let me know.

**Comment by Auroragan** [comment]
are these two nodes for normalization?

**Comment by skodippily** [comment]
Not quite sure about that. I cannot find any description of each layer and its task. 
It's a pre-trained model, and haven't modified anything on it. The only thing that I noticed is when I do the optimization, BatchNormalization layer will fuse to CONV layer.

**Comment by sun-xiangyu** [comment]
Please try [esp-ppq](https://github.com/espressif/esp-ppq) again.
